numpy/linalg/linalg.py

```diff
@@ -2357,14 +2357,13 @@ def multi_dot(arrays): return functools.reduce(np.dot, arrays)
     >>> # or
     >>> A.dot(B).dot(C).dot(D)
 
-
-    Example: multiplication costs of different parenthesizations
-    ------------------------------------------------------------
-
+    Notes
```

Comment:
Feel free to leave this in, since it's correct. Just add the backslashes on the lines you unwrapped, and this'll be good to merge